package.json

```diff
@@ -141,7 +141,7 @@
     "react-native-confirmation-code-field": "6.5.1",
     "react-native-default-preference": "1.4.1",
     "react-native-device-info": "^6.0.2",
-    "react-native-elements": "^1.2.7",
+    "react-native-elements": "2.3.2",
```

Comment:
I had to upgrade it because old version doesn't support custom `testID` property